launcher/ui/GuiUtil.cpp

```diff
@@ -75,10 +99,36 @@ std::optional<QString> GuiUtil::uploadPaste(const QString& name, const QString&
 
             if (response != QMessageBox::Yes)
                 return {};
+
+            if (baseUrl.toString() == "https://api.mclo.gs" && text.count("\n") > MaxMclogsLines) {
+                auto truncateResponse = CustomMessageBox::selectable(
+                                            parentWidget, QObject::tr("Confirm Truncate"),
+                                            QObject::tr("The log exceeds mclo.gs' limit: %1 lines (max %2).\n"
+                                                        "Prism can keep the first %3 and last %4 lines, trimming the middle.\n\n"
+                                                        "If you choose 'No', mclo.gs will only keep the first %2 lines, cutting off "
+                                                        "potentially useful info like crashes at the end.\n\n"
+                                                        "Proceed with Prism's truncation?")
```

Comment:
```suggestion
                                            QObject::tr("The log has %1 lines, exceeding the limit of %2.\n"
                                                        "The launcher can keep the first %3 and last %4 lines, trimming the middle.\n\n"
                                                        "If you choose 'No', mclo.gs will only keep the first %2 lines, cutting off "
                                                        "potentially useful info like crashes at the end.\n\n"
                                                        "Proceed with truncation?")
```
Replaced with The launcher to make things easier for forks; if we really need the application name there is a constant for that but I don't think we do here.

Reply:
that's fair, wouldn't it make sense to mention mclo.gs in the first line though? to make it clear 25k line limit is set by mclo.gs

Reply:
That was my mistake, updated